pkg/sidecar/appclone.go

```diff
@@ -157,12 +226,13 @@ func cloneFromSource(cfg *Config, host string) error {
 		return err
 	}
 
+	cloneSucceeded = true
 	return nil
 }
 
-func xtrabackupPreperData() error {
+func xtrabackupPrepareData() error {
 	// nolint: gosec
-	xtbkCmd := exec.Command("xtrabackup", "--prepare",
+	xtbkCmd := exec.Command(xtrabackupCommand, "--prepare",
```

Comment:
By replacing these with a variable, I can disable them as needed in unit tests. For example, when we pass a valid xbstream which is not necessarily also a valid xtrabackup directory.